src/main/scala/tilelink/Broadcast.scala

```diff
@@ -38,33 +132,38 @@ class TLBroadcast(params: TLBroadcastParams)(implicit p: Parameters) extends Laz
     clientFn  = { cp =>
       cp.v1copy(clients = Seq(TLMasterParameters.v1(
         name     = "TLBroadcast",
+        /* Add 2 bits encoded in [[TLBroadcastConstants]] to record transaction type for each sourceId in clients. */
         sourceId = IdRange(0, 1 << log2Ceil(cp.endSourceId*4)))))
     },
     managerFn = { mp =>
       mp.v1copy(
         endSinkId  = params.numTrackers,
         managers   = mp.managers.map { m =>
-          // We are the last level manager
+          /** This is the last level coherence manager, so it cannot be probed by mangers. */
```

Comment:
"outward managers"

Reply:
nit: can you use the `//` style comments for those that are not explicitly scaladoc for something? That is the style adopted in the code base

Reply:
@mwachs5 Sure I’ll change it back:)
I’ll change all my modifications with `/** xx */` for scala doc and `//` for comments.